recipes/rangeless/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+class RangelessConan(ConanFile):
+    name = "rangeless"
+    description = "c++ LINQ -like library of higher-order functions for data manipulation"
+    license = "Public domain"
```

Comment:
🤯 

https://github.com/ast-al/rangeless/blob/master/LICENSE

https://www.ncbi.nlm.nih.gov/home/about/policies/

😕

Reply:
(Not a lawyer)
Now I read it the second time, I think this is basically MIT (public domain redistribution rights + requirements to cite the source). The author just happen to work for NCBI, so the organization name is mentioned.
Though that means it's not "Public domain" anymore. What would be the appropriate name to put here?

Reply:
I haven't a clue what we should do...

```suggestion
    license = "rangless-public-domain"
```

@uilianries when you have a change, what should the license be?

Reply:
I agree to MIT.

CC0 anyone can re-license on any terms, and license copy is not mandatory. As that license requires citation about NCBI, it fits better for MIT.

https://choosealicense.com/licenses/cc0-1.0/
https://choosealicense.com/licenses/mit/

Reply:
Thank you for your input!

Reply:
I would love to know what NCBI thinks about, I'll send an email asking about and updating here when having an answer.

Reply:
Well, I didn't find any email address, only center support which is a FAQ actually.